src/Events/Integrations/Plugins/Elementor/Template/Controller.php

```diff
@@ -81,71 +91,34 @@ public function add_filters(): void {
 	 */
 	public function remove_filters(): void {
 		remove_filter( 'tribe_events_template_single-event.php', [ $this, 'filter_override_event_template' ] );
-		remove_filter( 'get_post_metadata', [ $this, 'bind_meta_courier' ], 25 );
-		remove_filter( 'elementor/document/config', [ $this, 'bind_content_handler' ], 25 );
-	}
-
-	/**
-	 * Handle the content of the Event for better compatibility.
-	 *
-	 * @param array $config The additional document configuration.
-	 * @param int   $id     The post ID of the document.
-	 */
-	public function bind_content_handler( $config, $id ): array {
-		tribe( Content::class )->save_post_content_as_filtered_content( $id );
-
-		return (array) $config;
 	}
 
 	/**
-	 * Binds the Meta Courier to copy data our Document type to the Post in question.
+	 * Include the template selection helper.
 	 *
 	 * @since TBD
 	 *
-	 * @param mixed  $value  The value to return, either a single metadata value or an array
-	 *                       of values depending on the value of `$single`. Default null.
-	 * @param int    $id     ID of the object metadata is for.
-	 * @param string $key    Metadata key.
-	 *
-	 * @return mixed
+	 * @return void
 	 */
-	public function bind_meta_courier( $value, $id, $key ) {
-		// Prevents the clone of the Base template from happening all the time.
-		if ( ! empty( $value ) ) {
-			return $value;
-		}
-
-		if ( ! is_string( $key ) ) {
-			return $key;
-		}
+	public function include_template_selection_helper(): void {
+		$preview = tribe_get_request_var( 'elementor-preview' );
```

Comment:
Reasons for using this vs `Plugin::$instance->preview->is_preview_mode()`? 
More reliable?

Reply:
Yeah more reliable.